cpp/src/io/utilities/config_utils.cpp

```diff
@@ -82,4 +84,12 @@ bool is_stable_enabled() { return is_all_enabled() or get_env_policy() == usage_
 
 }  // namespace nvcomp_integration
 
+namespace kvikio_setting {
+void set_thread_pool_nthreads_from_env()
+{
+  auto nthreads = getenv_or<unsigned int>("KVIKIO_NTHREADS", 8U);
+  kvikio::defaults::thread_pool_nthreads_reset(nthreads);
```

Comment:
need to make sure we only call this once.

Reply:
Good idea. Completed.